lib/rubycritic/core/smell.rb

```diff
@@ -47,7 +48,16 @@ def to_json(*a)
     end
 
     def doc_url
-      "https://github.com/troessner/reek/blob/master/docs/#{dasherized_type}.md"
+      case analyser
+      when 'reek'
+        "https://github.com/troessner/reek/blob/master/docs/#{dasherized_type}.md"
+      when 'flay'
+        'http://docs.seattlerb.org/flay/'
```

Comment:
Let's put this string and the next into constants at the top of the file (including ".freeze")